src/frontend/screens/Settings/index.tsx

```diff
@@ -68,41 +63,24 @@ function Settings() {
   // Load Heroic's or game's config, only if not loaded already
   useEffect(() => {
     const getSettings = async () => {
-      const config = isDefault
-        ? await window.api.requestAppSettings()
-        : await window.api.requestGameSettings(appName)
+      const config = await window.api.requestAppSettings()
       setCurrentConfig(config)
-
-      if (!isDefault) {
-        setTitle(gameInfo?.title ?? appName)
-      } else {
-        setTitle(t('globalSettings', 'Global Settings'))
-      }
     }
     getSettings()
-  }, [appName, isDefault, i18n.language])
-
-  // generate return path
-  let returnPath = '/'
-  if (!fromGameCard) {
-    returnPath = `/gamepage/${runner}/${appName}`
-    if (returnPath.includes('default')) {
-      returnPath = '/library'
-    }
-  }
+  }, [])
 
   // create setting context functions
   const contextValues: SettingsContextType | null = useSettingsContext({
-    appName,
-    gameInfo,
-    runner
+    appName
   })
 
   // render `loading` while we fetch the settings
-  if (!title || !contextValues) {
+  if (!currentConfig || !contextValues) {
```

Comment:
we were checking `title` here, but title doesn't really change, it's better to check for `currentConfig` since it's what the comment is saying originally and that's the real intention